index.html

```diff
@@ -97,86 +103,129 @@ <h1>bam2.iobio</h1>
+
+            <iobio-percent-box percent-key="proper_pairs" total-key="total_reads"></iobio-percent-box>
           </iobio-panel>
 
           <iobio-panel>
-            <iobio-percent-box
-              label="Singletons"
-              percent-key="singletons"
-              total-key="total_reads">
-            </iobio-percent-box>
+            <iobio-info-button label="Singletons">
+              <div slot="content">
+                <p>This is custom content for the Singletons percent box.</p>
+              </div>
+            </iobio-info-button>
+
+            <iobio-percent-box percent-key="singletons" total-key="total_reads"></iobio-percent-box>
           </iobio-panel>
 
           <iobio-panel>
-            <iobio-percent-box
-              label="Both Mates Mapped"
-              percent-key="both_mates_mapped"
-              total-key="total_reads">
-            </iobio-percent-box>
+            <iobio-info-button label="Both Mates Mapped">
+              <div slot="content">
+                <p>This is custom content for the Both Mates Mapped percent box.</p>
+              </div>
+            </iobio-info-button>
+
+            <iobio-percent-box percent-key="both_mates_mapped" total-key="total_reads"></iobio-percent-box>
           </iobio-panel>
          
           <iobio-panel>
-            <iobio-percent-box
-              label="Duplicates"
-              percent-key="duplicates"
-              total-key="total_reads">
-            </iobio-percent-box>
+            <iobio-info-button label="Duplicates">
+              <div slot="content">
+                <p>This is custom content for the Duplicates percent box.</p>
+              </div>
+            </iobio-info-button>
+
+            <iobio-percent-box percent-key="duplicates" total-key="total_reads"></iobio-percent-box>
           </iobio-panel>
 
         </div>
 
         <div class='histogram-column'>
           <iobio-panel>
-            <iobio-histogram
-              label="Read Coverage Distribution"
-              broker-key="coverage_hist">
-            </iobio-histogram>
+            <iobio-info-button label="Read Coverage Distribution">
+              <div slot="content">
+                <p>This is custom content for the Read Coverage Distribution histogram.</p>
+              </div>
+            </iobio-info-button>
+
+            <iobio-histogram broker-key="coverage_hist"></iobio-histogram>
           </iobio-panel>
 
           <iobio-panel>
-            <iobio-tabs label-1="Fragment Length" label-2="Read Length">
-              <iobio-histogram
-                broker-key="frag_hist"
-                ignore-outliers>
-              </iobio-histogram>
-
-              <iobio-histogram
-                broker-key="length_hist"
-                ignore-outliers>
-              </iobio-histogram>
+            <iobio-tabs>
+              <iobio-tab slot="tab">
+                <iobio-info-button modal-header="Fragment Length">
```

Comment:
That's only one of the reasons. I use different attribute names because when we use Tabs-TabPanel pattern to wrap the chart, the Tab name will be used as the chart label. And here the user can decide whatever the tab name is. 

The attribute of `iobio-info-button` in Tabs would be only used as the modal header. For other charts, the attribute of 'iobio-info-button' would be used as the chart label and also the modal header. So you can see the code I wrote in info-button.js. They're different logic.

Reply:
The problem is the duplicated code. You have "Fragment Length" written twice, close to each other. Why can't you use `label` for both use cases?

Reply:
I tried changing the code to this and it looks fine:
```html
<iobio-tab slot="tab">
  <iobio-info-button label="Fragment Length">
    <div slot="content">
      <p>This is custom content for the Fragment Length histogram.</p>
    </div>
  </iobio-info-button>
</iobio-tab>
```

Only problem is click events on "Fragment Length" aren't working, but that seems fixable

Reply:
This layout helps me understand what you want for the structure. Do you mean that you want to use the label attribute to replace the tab name?

Reply:
Yes and no. Basically it would be nice if we could put whatever content we want in the tabs. Doesn't necessarily need to be words. Maybe we just want to do a div square with a background color. In this case we're using the info button, which happens to have text, but the tab shouldn't need to be aware of that.